test/plugin.spec.js

```diff
@@ -93,7 +93,7 @@ describe('Babel Root Import - Plugin', () => {
   });
 
   it('supports re-export syntax (export * from ... or export { named } from ...)', () => {
-    const targetRequire = slash(`${process.cwd()}/some/example.js`);
+    const targetRequire = slash('/some/example.js');
     const transformedExportNamed = babel.transform("export * from '@/some/example.js';", {
```

Comment:
The issue was already there but I fixed it anyway, I also added a couple of comments to the code because its not that obvious why I left each line there